gizmos/tree.py

```diff
@@ -2,6 +2,8 @@
 import sqlite3
 import sys
 
+import gizmos.hiccup as gh
```

Comment:
This is fine, but I think I would prefer `from gizmos.hiccup import curie2href, render`